cms/navigation/forms.py

```diff
@@ -131,15 +136,15 @@ def validate_sub_link_wrapper(
             return sec_errors
 
         def validate_column_block(
-            column_block: Any,
+            column_block: StructValue,
             col_idx: int,  # pylint: disable=unused-argument
         ) -> list[StructBlockValidationError]:
             """Validate a single column block, including its sections."""
             errors: list[StructBlockValidationError] = []
             column_value = column_block.value
             sections = column_value.get("sections", [])
 
-            def validate_section_wrapper(sec_data: Any, sec_idx: int) -> list[StructBlockValidationError]:
+            def validate_section_wrapper(sec_data: StructValue, sec_idx: int) -> list[StructBlockValidationError]:
```

Comment:
kudos for the updated type hints

Reply:
Yes, as I was a bit lazy when I was doing the `forms.py` code for the main menu so knew type hinting wasn't great, so we took the opportunity to improve this time =)